src/error/exporterrors.cpp

```diff
@@ -113,7 +113,29 @@ ostream & operator<< (ostream & os, parse_t & p)
 		os << p[i]["macro"] << " " << i << endl;
 	}
 
+    os << endl << endl;
+
+	/* Temporarily disable enum generation.
```

Comment:
Please revert that change, there is little use of passing all enums to the user.